lib/braintree.js

```diff
@@ -34,9 +34,19 @@ module.exports = class {
     const writePromise = async isMin => new Promise((resolve, reject) => {
       const { min, max } = redshiftTimestamps;
 
-      const searchStream = this.client.transaction.search(search => (
-        isMin ? search.createdAt().min(max) : search.createdAt().max(min)
-      ));
+      var searchStream;
+
+      if (responseObject == 'disputes') {
+        searchStream = this.client.dispute.search(search => (
+          isMin ? search.receivedDate().min(max) : search.receivedDate().max(min)
```

Comment:
Braintree API doesn't allow searching disputes by createdAt() for some reason, so this searches by receivedDate() instead, which seems like the closest searchable field.